examples/03-maps/plot_sumMap.py

```diff
@@ -0,0 +1,192 @@
+"""
+Maps: ComboMaps
+===============
+
+Invert synthetic magnetic data with variable background values
+and a single block anomaly buried at depth. We will use the Sum Map
+to invert for both the background values and an heterogeneous susceptibiilty
+model.
+
+.. code-block:: python
+    :linenos:
+
+
+"""
+from SimPEG import (
+    Mesh, Utils, Maps, Regularization,
+    DataMisfit, Optimization, InvProblem,
+    Directives, Inversion, PF
+)
+import numpy as np
+import matplotlib.pyplot as plt
+
+
+def run(plotIt=True):
```

Comment:
Thanks for adding this @fourndo! Rather than using the `def run`, would you mind turning this into a notebook styled example (e.g. http://discretize.simpeg.xyz/en/latest/auto_examples/plot_slicer_demo.html#sphx-glr-auto-examples-plot-slicer-demo-py, the source is: https://github.com/simpeg/discretize/blob/master/docs/examples/plot_slicer_demo.py). 

The notebook-styled examples are a bit more friendly for new users